packages/flutter_driver/test/src/real_tests/flutter_driver_test.dart

```diff
@@ -24,10 +27,107 @@ const String _kWebScriptSuffix = "')";
 
 void main() {
   final List<String> log = <String>[];
+
   driverLog = (String source, String message) {
     log.add('$source: $message');
   };
 
+  group('VMServiceFlutterDriver with logCommunicationToFile', () {
+    late FakeVmService fakeClient;
+    FakeVM fakeVM;
+    late FakeIsolate fakeIsolate;
+    late VMServiceFlutterDriver driver;
+    int driverId = 0;
+    setUp(()  async {
+      _cleanTestOutputFiles();
+      fakeIsolate = FakeIsolate();
+      fakeVM = FakeVM(fakeIsolate);
+      fakeClient = FakeVmService(fakeVM);
+      fakeClient.responses['waitFor'] = makeFakeResponse(<String, dynamic>{'status':'ok'});
+    });
+
+    tearDown(() {
+      driverId++;
+    });
+
+    group('logCommunicationToFile', (){
+        test('logCommunicationToFile = true', () async {
+        // Given
```

Comment:
While the structure these comments imply is good, it isn't necessary to leave them in the test

Reply:
I removed.